hybridse/src/planv2/ast_node_converter.cc

```diff
@@ -2158,17 +2158,35 @@ static const absl::flat_hash_map<std::string_view, ShowTargetInfo> showTargetMap
     {"JOBLOG", {node::CmdType::kCmdShowJobLog, true}},
 };
 
+static const absl::flat_hash_map<std::string_view, node::ShowStmtType> SHOW_STMT_TYPE_MAP = {
+    {"JOBS", node::ShowStmtType::kJob},
+};
+
 base::Status convertShowStmt(const zetasql::ASTShowStatement* show_statement, node::NodeManager* node_manager,
                              node::SqlNode** output) {
     CHECK_TRUE(nullptr != show_statement && nullptr != show_statement->identifier(), common::kSqlAstError,
                "not an ASTShowStatement")
 
     auto show_id = show_statement->identifier()->GetAsStringView();
 
+    if (auto iter = SHOW_STMT_TYPE_MAP.find(absl::AsciiStrToUpper(show_id));
```

Comment:
looks strange. Output node type should unified

Reply:
will move all show statement from `CmdNode` to `ShowStmt` later.